packages/b/boost/xmake.lua

```diff
@@ -138,11 +138,18 @@ package("boost")
 
         -- get msvc
         local msvc
+        local is_clang_tc = false
         if package:is_plat("windows") then
-            msvc = toolchain.load("msvc", {plat = package:plat(), arch = package:arch()})
+            -- Boost's bootstrap.bat needs the runenvs from the toolchain to locate cl
+            local tc_name = package:config("toolchains") or "msvc"
```

Comment:
please use always use msvc toolchain, msvc and clang-cl has same runenvs. and the windows platform is currently strictly bound to the msvc toolchain now.

Reply:
Ok. I can close the PR and keep a local version of the boost package. As I said in a past comment, when using the clang-cl toolchain, `msvc:runenvs()` is empty so the boost bootstrap fails because it can't find the cl command to build b2 with.

Reply:
try it again.

Reply:
I've tried it. It doesn't seem to work when using the default auto-detected toolchain...

Reply:
I notice the pipelines also fail with that same error

Reply:
Thinking about this a little bit, can xmake provide defaulted configs to packages? CMake for example compiles a list of all the used variables in a CMakeCache.txt file (including the auto-detected ones). I've noticed xmake only provides the explicitly defined configurations within many of its APIs.